app/controllers/foreman_bootdisk/api/v2/disks_controller.rb

```diff
@@ -7,10 +9,10 @@ class DisksController < ::Api::V2::BaseController
         include ::Api::Version2
 
         resource_description do
-          api_base_url "/bootdisk/api"
+          api_base_url '/bootdisk/api'
```

Comment:
Honestly I hate this cop, Ruby parser should be able to figure it itself before bytecode compilation. I don't mind keeping all the changes as is but can you please disable this pesky one?

Reply:
I love the cop. I get headaches when I need to read `"no variables here"`.

Reply:
Normally I'd insist, but since it's you, I buy it. ;-)